config/storybook-preset-carbon/index.js

```diff
@@ -35,6 +37,36 @@ module.exports = {
   ].filter(Boolean),
 
   webpack(config) {
+    const sassLoader = {
+      loader: 'sass-loader',
+      options: {
+        prependData() {
```

Comment:
I think this method was changed to `additionalData` in v10 of `sass-loader

Reply:
@joshblack The config from your initial work on the previous PR also includes a `content` param

https://github.com/carbon-design-system/carbon/pull/8228/commits/dedd96d64a9334227e66c9892b760683446b389d